recipes/conversation_assistant/conversation.js

```diff
@@ -25,15 +25,18 @@ const assistant = new AssistantV2({
 });
 
 // these are the hardware capabilities that TJ needs for this recipe
-const hardware = [TJBot.HARDWARE.MICROPHONE, TJBot.HARDWARE.SPEAKER, TJBot.HARDWARE.LED_NEOPIXEL, TJBot.HARDWARE.SERVO];
+// const hardware = [TJBot.HARDWARE.MICROPHONE, TJBot.HARDWARE.SPEAKER, TJBot.HARDWARE.LED_NEOPIXEL, TJBot.HARDWARE.SERVO];
+const hardware = [TJBot.HARDWARE.MICROPHONE, TJBot.HARDWARE.SPEAKER]
```

Comment:
don't we need the LED and servo? it looks like we support those as actions…

Reply:
Yep! I forgot to uncomment the original